alf/experience_replayers/replay_buffer.py

```diff
@@ -568,6 +568,9 @@ def gather_all(self):
         """
         size = self._current_size.min()
         max_size = self._current_size.max()
+        if debug:
+            print(f'size={size}')
```

Comment:
We can use logging.info to replace print for better message.

Reply:
I removed those two lines. Did not intend to keep them here. Sorry!